modules/messages/server/controllers/messages.server.controller.js

```diff
@@ -280,6 +282,13 @@ exports.send = function(req, res) {
       });
     },
 
+    // Here we collect some message data into the influxdb
+    function (message, done) {
+      // the module returns a promise
+      messageToInfluxService(message, { Message: Message, influxService: influxService })
```

Comment:
partially fixed with f587c8f6eb377cf93fb3fac050e5bf581844d611

still didn't figure out how to properly load mongoose models (outside of the app)